### ISSUE TITLE
WCAG 1.4.3: Elements must have sufficient color contrast (.ribbon)

### ISSUE THREAD

**Issue by jwunderl** [microsoft/pxt-microbit#2299]
#### Issue
`Elements must have sufficient color contrast` ([color-contrast](https://dequeuniversity.com/rules/axe/3.2/color-contrast?application=msftAI))
#### Target application
[Microsoft MakeCode for micro:bit](https://makecode.microbit.org/#editor)
#### Element path
.ribbon
#### Snippet
`<label class="ui orange right ribbon label">Local</label>`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 2.34 (foreground color: #ffffff, background color: #ff8b27, font size: 10.3pt, font weight: bold). Expected contrast ratio of 4.5:1
#### Environment
Chrome version 75.0.3770.142
This accessibility issue was found using Accessibility Insights for Web 2.7.1 (axe-core 3.2.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).

local ribbon on extensions screen

![Screen Shot 2019-07-18 at 3 28 53 PM](https://user-images.githubusercontent.com/5615930/61496349-c6e77680-a970-11e9-9a49-bbd660d365b2.png)
